### ISSUE TITLE
[BUG] Old versions fail to start on any other renderer than zink/vulkan

### ISSUE THREAD

**Issue by LolexIsALazyPotato** [PojavLauncherTeam/PojavLauncher#2698]
### Describe the bug

vgpu or any gl4es renderer crashes after a couple seconds in game.

### The log file and images/videos

https://user-images.githubusercontent.com/95535264/152683372-0531a106-385a-457f-94ee-768ed1f75e1c.mp4

I picked vgpu there, but I can guarantee you the same happens if I pick any gl4es renderer. Edited to zoom in, and compress the file to 10mb. There is also no log file.

### Steps To Reproduce

```markdown
1. Start pojav launcher
2. Pick any render other than vulkan/zink.
3. Start up Minecraft 1.0 (some later and earlier versions do this too)
4. Enter a world.
5. The game crashes without a crash log, or anything else after a couple seconds in the world.
```

### Expected Behavior

World runs properly, lag free.

### Platform

```markdown
- Device model: Samsung M51 6/128G
- CPU architecture: aarch64
- Android version: 11
- PojavLauncher version: v3
```

**Comment by artdeell** [comment]
1. Outdated version of the launcher. Use the latest builds from the Actions tab.
2. Enable "Force openGL 1" in experimental settings in the launcher

**Comment by LolexIsALazyPotato** [comment]
Tried it and no matter what I do, it just comes up with this and doesn't download the app. It happens both with the runtime included and the version without runtime included. I'm not sure if I downloaded the right thing, but these were the only apk's there.
![Screenshot_20220206-221749_Package installer](https://user-images.githubusercontent.com/95535264/152701803-62ee8a6c-e139-4431-b7aa-055dc64d3143.png)
